### PR TITLE
Image Processing in Python with Pillow

### PR DESCRIPTION
**Pull request by echessa**

### CODE REVIEW

File: _posts/2017-04-14-image-processing-in-python-with-pillow.markdown [hunk 1]
```diff
@@ -0,0 +1,649 @@
+---
+layout: post
+title: Image Processing in Python with Pillow
+description: Learn how to process images using the Pillow Python library.
+date: 2017-04-14 17:54
+category: Technical Guide, Python
+banner:
```
Comment (reviewgen) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: Let's remove this. This is specific to Authentication or you can actually change it to:

"Auth0 makes it easy to add authentication to your Python application."

File: _posts/2017-04-14-image-processing-in-python-with-pillow.markdown [hunk 2]
```diff
@@ -0,0 +1,649 @@
...
+  image: https://pbs.twimg.com/profile_images/510760404411109380/wDGjWJxk.png
+tags: 
+- python
+- pillow
+---
+
+## Introduction
+A lot of applications use digital images, and with this there is usually a need to process the images used. If you are building your application with Python and need to add image processing features to it, there are various libraries you could use. Some popular ones are [OpenCV](https://pypi.python.org/pypi/opencv-python), [scikit-image](http://scikit-image.org/), [Python Imaging Library](http://www.pythonware.com/products/pil/) and [Pillow](https://python-pillow.org/).
+
+We won't debate on which library is the best here, they all have their merits. This article will focus on Pillow, a library that is powerful, provides a wide array of image processing features and is simple to use.
```
Comment (reviewgen) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: add comma between features and simple

File: _posts/2017-04-14-image-processing-in-python-with-pillow.markdown [hunk 3]
```diff
@@ -0,0 +1,649 @@
...
+## Installation and Project Setup
+Before installing Pillow, there are some prerequisites that must be satisfied. These vary for different operating systems. We won't list the different options here, you can find the prerequisites for your particular OS in this [installation guide](https://pillow.readthedocs.io/en/3.0.0/installation.html#os-x-installation).
+
+After installing the prerequisite libraries, you can install Pillow with `pip:
+
+```sh
+$ pip install Pillow
+```
+
+To follow along, you can [download](https://github.com/echessa/misc/raw/master/rp/img_pro/images.zip) the images (coutesy of [unsplash](https://unsplash.com/)) that we'll use in the article. You can also use your own images.
```
Comment (reviewgen) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: unsplash => Unsplash

File: _posts/2017-04-14-image-processing-in-python-with-pillow.markdown [hunk 4]
```diff
@@ -0,0 +1,649 @@
...
+      <p>You can <a href="{{ url_for('upload') }}">upload images</a> or head over to the <a href="{{ url_for('gallery') }}">gallery</a></p>
+      <p><a href="{{ url_for('logout') }}">Logout</a></p>
+    {% else %}
+      <p><a href="#" class="login">Login</a> to upload images or head over to the <a href="{{ url_for('gallery') }}">gallery</a></p>
+    {% endif %}
+    {% endraw %}
+
+Run the app and you should now be able to log out.
+
+## Conclusion
```
Comment (reviewgen) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: I would just add a final sentence asking users to sign up for Auth0 if they are building Python application to make the process of authentication easy for them. :)

File: _posts/2017-04-14-image-processing-in-python-with-pillow.markdown (hunk 2 above)
Comment (echessa) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: Done

File: _posts/2017-04-14-image-processing-in-python-with-pillow.markdown (hunk 3 above)
Comment (echessa) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: Fixed

File: _posts/2017-04-14-image-processing-in-python-with-pillow.markdown (hunk 4 above)
Comment (echessa) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: I've added this in the Conclusion

File: _posts/2017-04-14-image-processing-in-python-with-pillow.markdown (hunk 1 above)
Comment (echessa) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: @reviewgen @kukicado I didn't get this. Should I put `Auth0 makes it easy to add authentication to your Python application.` as the `description`? I think you might have pointed to the wrong position. Where should I make the change?

Comment (echessa) [inline _posts/2017-04-14-image-processing-in-python-with-pillow.markdown]: Fixed

**Pull request merged**